circle.yml

```diff
@@ -6,6 +6,6 @@ test:
   pre:
     - npm run eslint
   override:
-    - npm run istanbul
+    - npm test
```

Comment:
I thought so too. But in that case if we're running both `eslint` and `jest` as part of `npm test` then we should remove the `pre` command too, so that we're not running `eslint` twice.